onnxruntime_cpp_samples/README.md

```diff
@@ -3,3 +3,15 @@
 To run the samples simply run `cmake -Bbuild . && cmake --build build` inside this folder.
 Afterwards the applications can be executed from this directory using `./build/cuda_provider/cuda_sample` and `./build/dml_provider/dml_sample` without any command line arguments. 
 For any further information please consult the README files in the respective subdirectories.
+
+
+## Dependencies 
+
+- [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads)
+
+Verify which version of onnxruntime is compatible with that CUDA version at [NVIDIA - CUDA | onnxruntime website](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements).Then update the `onnxruntime_INSTALL` inside the `CMakeLists.txt` or install the CUDA Toolkit that is compatible with the onnxruntime version that the sample is using.
+
```

Comment:
Indeed better wording, I was trying to be as generic as possible in case you or someone in the future update the `onnxruntime_INSTALL` to a newer version of the onnxruntime, and the documentation needs to be updated as well.